briefcase-api.rst

```diff
@@ -31,9 +31,9 @@ Push
 
 Interfaces used during Push actions.
 
-1. *formUpload* to upload a form and its media files to ODK Aggregate.
+1. *form upload* to upload a form and its media files to ODK Aggregate.
```

Comment:
I'm not 100% sure that the API requires it, but to be safe, it should be case-sensitive. This change should be reverted.